frontend/src/views/ProductDetails.vue

```diff
@@ -46,12 +45,22 @@ function printQr () {
 
   <div class="row box bg-white border-top">
     <text-input name="Name" v-model:value="product.name" @update:value="save"/>
-    <number-input name="Stock" v-model:value="product.stock" @update:value="save"/>
+    <div class="d-flex align-items-center p-2 border-bottom">
```

Comment:
Zou dit naar een appart input componentje kunnen? 

En denk dat het mischien handig is om het plus en min knopje te laten staan voor wanneer je op desktop snel iets toe wilt voegen